cert/spec.go

```diff
@@ -432,7 +434,12 @@ func (spec *Spec) EnforcePKI(enableActions bool) error {
 			if err != nil {
 				log.Debugf("spec %s: key failed to be read: %s", spec, err)
 			} else {
-				err = spec.checkDiskPKI(existingCert, keyData)
+				err = verifyCertChain(currentCA, existingCert)
+				if err != nil {
+					log.Debugf("spec %s: CA has changed, cert is no longer valid via it: %s", spec, err)
```

Comment:
Should this be more than a debug level, if a cert has suddenly become invalid? Perhaps info?

Reply:
debug is at this layer, but the error returned gets logged via info.  I explicitly added debug for these sets of functions so any screwup in a higher layer could be isolated via loglines in debug mode.